Meta/build-manpages-website.sh

```diff
@@ -106,7 +106,7 @@ pandoc -f gfm -t html5 -s \
 echo 'Copying images'
 rsync -a Meta/Websites/man.serenityos.org/banner.png output/ &
 rsync -a Base/usr/share/man/man7/LibDSP_classes.svg output/ &
-rsync -a Base/usr/share/man/man1/*.png output/ &
+find Base/usr/share/man/ -iname '*.png' -exec rsync -a {} output/ \; &
```

Comment:
what's `\;` for?

Reply:
I think it terminates `-exec`; find does not run without it.